Cargo.toml

```diff
@@ -47,6 +47,7 @@ warp = "0.3.2"
 tracing = "0.1.35"
 tracing-subscriber = { version = "0.3.15", features = ["json"] }
 prometheus = { version = "0.13.0", default-features = false }
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
required, else run into following cross compile error for openssl: `run pkg_config fail: "pkg-config has not been configured to support cross-compilation.\n\nInstall a sysroot for the target platform and configure it via\nPKG_CONFIG_SYSROOT_DIR and PKG_CONFIG_PATH, or install a\ncross-compiling wrapper for pkg-config and set it via\nPKG_CONFIG environment variable."`

Reply:
This seems to make the most sense I guess 
https://github.com/messense/rust-musl-cross/issues/35